File: insertion_sort/Python/insertion_sort.py
# Python program for implementation of Insertion Sort 
  
# Function to do insertion sort 
def insertion_Sort(arr): 
  
    # Traverse through 1 to length of array 
    for i in range(1, len(arr)): 
        key = arr[i]
        j = i-1
        while j >= 0 and key < arr[j] : 
            arr[j + 1] = arr[j] 
            j -= 1
        arr[j + 1] = key 

# Driver code to test above 
arr = [12, 11, 13, 5, 6] 
insertion_Sort(arr) 
for i in range(len(arr)): 
    print ("% d" % arr[i]) 

File: insertion_sort/julia/insertionsort.jl
function insertionsort(lis)
    for item in 2:length(lis)
        key = lis[item]
        i = item - 1
        while i > 0 && key < lis[i]
            lis[i+1] = lis[i]
            i = i - 1
        end
        lis[i+1] = key
    end
    return lis
end
